tests/unit/utils/initPlayer.js

```diff
@@ -5,6 +5,8 @@ module.exports = (attrs) => {
         element: container,
         attrs: attrs || {}
     });
+
+    player.__video = player.__video || document.createElement("video");
```

Comment:
Hi @sambua, if we remove the `init` function from TrackTags' constructor we can test `generateVTTFromObject` without needing this workaround.

```suggestion
```

Reply:
Hi @ntshcalleia actually I had plan test as possible all events here on unit test without running server. So there was more other tests, but because of one event related to track tags not triggered I had to move all of them to playwright side. I left it as it is in the future could be require some cases when video element could be require.